### PR TITLE
chore: update unenv dependency version

### PR DESCRIPTION
**Pull request by vicb**
## What this PR solves / how to test

Fixes https://github.com/cloudflare/workerd/issues/2720

By pulling in https://github.com/unjs/unenv/pull/310

Fixes N/A (the issue is on the worked repo)

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [x] Tests included
  - [ ] Tests not necessary because: unenv update
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [x] Required
  - [ ] Not required because:
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Changeset included
  - [ ] Changeset not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [x] Documentation not necessary because: dep update

### CODE REVIEW

**Review by petebacondarwin — DISMISSED** [review]
Can you add the code that was broken to this file fixtures/nodejs-hybrid-app/src/index.ts to provide it was fixed.

image

**Comment by vicb** [comment]
> Can you add the code that was broken to this file fixtures/nodejs-hybrid-app/src/index.ts to provide it was fixed.
> 
> image

I'll do but - I don't think this test belongs here vs unenv but I get that it's ok until we have a better test story for unenv.

File: fixtures/nodejs-hybrid-app/tests/index.test.ts [hunk 1]
```diff
@@ -48,4 +48,19 @@ describe("nodejs compat", () => {
 			await stop();
 		}
 	});
+
+	test("crypto.X509Certificate is implemented", async ({ expect }) => {
```
Comment (petebacondarwin) [inline fixtures/nodejs-hybrid-app/tests/index.test.ts:52]: NIT the other tests use `it()`; did you switch to `test()` for a specific reason?

**Review by petebacondarwin — APPROVED** [review]

File: fixtures/nodejs-hybrid-app/tests/index.test.ts (hunk 1 above)
Comment (vicb) [reply fixtures/nodejs-hybrid-app/tests/index.test.ts:52]: "test crypto.X509Certificate is implemented" sounds better that "it crypto.X509Certificate is implemented"
but no strong opinion, I can it it if you prefer.

Comment (petebacondarwin) [reply fixtures/nodejs-hybrid-app/tests/index.test.ts:52]: No worries. Just interested

**Comment by vicb** [comment]
Thanks!

**Pull request merged**